kolibri/deployment/default/settings/base.py

```diff
@@ -195,5 +195,10 @@
 
 AUTH_USER_MODEL = 'kolibriauth.DeviceOwner'
 
+AUTH_ANONYMOUS_USER_MODEL = "kolibri.auth.models.KolibriAnonymousUser"
```

Comment:
Would be good to document why a new setting is necessary, at least with a comment. Otherwise we risk building a file full of opaque settings. As a convention it might be helpful also to indicate that this is Kolibri-specific -- at first I thought it might be a Django setting.